README.md

```diff
@@ -15,6 +15,10 @@ L.tileLayer.provider('Stamen.Watercolor').addTo(map);
 Leaflet-providers tries to use `https://` if the page uses `https://` and the provider supports it.
 You can force the use of `http://` by passing `force_http: true` in the options argument.
 
+## Retina tiles
+
+Some providers have retina tiles, for which the URL only needs to be slightly adjusted, e.g. @2x.png. For this, add the retina option in the URL, e.g. {retina}.png, and set a retina value in the options, e.g. retina: '@2x'. If Leaflet's [detectRetina](http://leafletjs.com/reference.html#tilelayer-detectretina) is allowed and a retina screen is detected, the retina option will be activated in the URL.
+
```

Comment:
This last sentence is not really clear to me. Maybe something like this does better cover what actually happens:

If Leaflet detects a retina screen (`L.Browser.Retina` = `true`), the `retina` option passed to the tileLayer  is set to the value supplied, otherwise it's replaced by an empty string.